### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by tj-commits**
Clean up the code a little

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-var toString = {}.toString;
+var arrayObjectStringTag = '[object Array]';
+var toString = Object.prototype.toString;
 
 module.exports = Array.isArray || function (arr) {
-  return toString.call(arr) === '[object Array]';
+  return toString.call(arr) === arrayObjectStringTag;
```
Comment (ljharb) [inline index.js:7]: fwiw this is unlikely to be more performant (engines are very good at optimizing inline strings) and may even be slower.

**Comment by tj-commits** [comment]
It is actually faster.

https://jsperf.app/fapaco/1/preview

**Comment by ljharb** [comment]
Not in Safari:
Screenshot 2024-09-27 at 5 22 58 PM

**Comment by tj-commits** [comment]
Okay but for me `hi2` is faster

**Pull request closed without merging**